src/interfaces/IERC7540.sol

```diff
@@ -109,39 +126,42 @@ interface IERC7540Deposit {
 }
 
 interface IERC7540Redeem {
-    event RedeemRequest(address indexed sender, address indexed operator, address indexed owner, uint256 shares);
+    event RedeemRequest(address indexed sender, address indexed receiver, address indexed owner, uint256 shares);
 
     /**
-     * @dev Assumes control of shares from sender into the Vault and submits a Request for asynchronous redeem/withdraw.
-     * 
-     * - MUST support a redeem Request flow where the control of shares is taken from owner directly
+     * @dev Assumes control of shares from sender into the Vault and submits a Request for asynchronous redeem.
+     *
+     * - MUST support a redeem Request flow where the control of shares is taken from sender directly
      *   where msg.sender has ERC-20 approval over the shares of owner.
-     * - MUST revert if all of shares cannot be requested for redeem / withdraw.
-     * - sender MUST = msg.sender unless some unspecified explicit approval is given by the caller, approval of ERC7540 vault shares is NOT enough.
+     * - MUST revert if all of shares cannot be requested for redeem.
      *
-     * @param shares the amount of redemption shares from sender
+     * @param shares the amount of redemption shares to transfer from owner
```

Comment:
Imo, clearest to say "shares the amount of shares to be redeemed to transfer from owner" instead of introducing redemption